content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -184,7 +184,27 @@ the webhooks. There are three steps to complete the configuration.
   (yes, the same schema that's used by kubectl), so the field name is
   `kubeConfigFile`. Here is an example admission control configuration file:
 
+    {{< tabs name="admissionconfiguration_example1" >}}
+    {{% tab name="apiserver.config.k8s.io/v1" %}}
     ```yaml
+    apiVersion: apiserver.config.k8s.io/v1
```

Comment:
@liggitt , Looks like the new code block is not formatted. There is a known issue with Hugo (the version in use) and code blocks within list items. You may be able to fix by indenting the code block by 4 spaces?

Reply:
I looked at the formatting. If you align lines 187 - 223 to the left (instead of indenting), the backticks/yaml should be cleared up.
See the formatting for admission controllers as an example: https://deploy-preview-17548--kubernetes-io-vnext-staging.netlify.com/docs/reference/access-authn-authz/admission-controllers/#eventratelimit

Reply:
done